api/python/cellxgene_census/src/cellxgene_census/experimental/ml/pytorch.py

```diff
@@ -36,6 +37,10 @@
 The Tensors are rank 1 if ``batch_size`` is 1, otherwise the Tensors are rank 2."""
 
 
+# "Chunk" of X data, returned by each `Method` above
+ChunkX = Union[npt.NDArray[Any], sparse.csr_matrix]
```

Comment:
nit (optional): why `npt.NDArray[Any]` and not `npt.NDArray[np.number]` (or something even more specific?)

Reply:
Also, is the API signature really saying that it could return _any_ sparse type?  If it is actually constrained to a more specific set of sub-types (e.g, csr_matrix), it would be useful to further specify this type alias (even if just for the static type checking...)

Reply:
Hmm. This is opening up a big can of `typing` worms. There wouldn't be as much of an issue once we drop support for python 3.8.

I think I will go back to `Any` until we move to SPEC-0